extensions/variants.go

```diff
@@ -218,6 +222,20 @@ func parseFuncName(compoundName string) (name string, args ArgumentList) {
 	return name, args
 }
 
+func minArgumentCount(paramTypeList ArgumentList, variadicBehavior *VariadicBehavior) int {
+	if variadicBehavior == nil {
+		return len(paramTypeList)
+	}
+	return variadicBehavior.Min
+}
+
+func maxArgumentCount(paramTypeList ArgumentList, variadicBehavior *VariadicBehavior) int {
+	if variadicBehavior == nil {
+		return len(paramTypeList)
+	}
+	return variadicBehavior.Max
```

Comment:
shouldn't this be `len(paramTypeList) + variadicBehavior.Max`?

Reply:
fixed it